browser/misc_metrics/page_metrics_tab_helper.cc

```diff
@@ -25,13 +28,18 @@ PageMetricsTabHelper::PageMetricsTabHelper(content::WebContents* web_contents)
 
 PageMetricsTabHelper::~PageMetricsTabHelper() = default;
 
+void PageMetricsTabHelper::DidStartNavigation(
```

Comment:
perhaps you also want to check OnRedirectNavigation

Reply:
good catch, thanks